lib/fog/identity/openstack.rb

```diff
@@ -73,6 +73,8 @@ def initialize(options = {})
             @path = "/#{options[:openstack_identity_prefix]}/#{@path}"
           end
 
+          @path = "/#{options[:openstack_identity_prefix]}/" if options[:openstack_identity_prefix] == "v2.0"
```

Comment:
This change doesn't look good to me, when there is "v2.0" prefix, then path is dropped. Could you test some API calls (or ManageIQ refresh) first?

Reply:
Yes, you're right it should actually be:
```ruby
 @path = "/#{options[:openstack_identity_prefix]}/#{@path}" if options[:openstack_identity_prefix] == "v2.0"
```

Reply:
The first version is wrong, but that proposed fix seems to make the line identical to the conditional above-- ` @path = "/#{options[:openstack_identity_prefix]}/#{@path}"`